x-pack/plugins/security/server/authorization/privileges/feature_privilege_iterator/feature_privilege_iterator.test.ts

```diff
@@ -818,6 +826,133 @@ describe('featurePrivilegeIterator', () => {
     ]);
   });
 
+  it('excludes sub feature privileges when the minimum license is not met', () => {
```

Comment:
Optional nit: This test suite is rather large already, and this test case is exercising functionality that's encapsulated by `subFeaturePrivilegeIterator`. It may be better to split this test case out into a `sub_feature_privilege_iterator.test.tsx` instead.

Reply:
Yeah, I know you're right 😬

Reply:
I've opened https://github.com/elastic/kibana/issues/83457 to address this in a followup. I'm trying to reduce the number of tasks I have in flight, and would rather not hold this work up on that effort. I do think it's worthwhile though!